src/readers/src/main/scala/ImageReader.scala

```diff
@@ -48,9 +50,9 @@ object ImageReader {
            sampleRatio: Double = 1, inspectZip: Boolean = true): DataFrame = {
 
     val binaryRDD = BinaryFileReader.readRDD(path, recursive, spark, sampleRatio, inspectZip)
-    val binaryRDDlib = ImageSchema.loadLibraryForAllPartitions(binaryRDD, Core.NATIVE_LIBRARY_NAME)
+    //val binaryRDDlib = ImageSchema.loadLibraryForAllPartitions(binaryRDD, "opencv_core")
```

Comment:
Please remove commented-out code if it's not needed. If this is for reference, could you please add a more detailed explanation?